brave/brave-core
# Adding inline tip confirmation

Fixes: https://github.com/brave/brave-browser/issues/6151

<img width="1398" alt="Screen Shot 2019-10-24 at 1 03 45 AM" src="https://user-images.githubusercontent.com/8732757/67465805-bba08880-f5fa-11e9-8bc3-aecf801f276c.png">
<img width="1356" alt="Screen Shot 2019-10-24 at 1 03 27 AM" src="https://user-images.githubusercontent.com/8732757/67465806-bc391f00-f5fa-11e9-88fe-b510b2759cf2.png">

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Enable Rewards
2. Claim a grant
3. Go to a verified/connected publisher and leave a one-time tip
4. Confirm that the updated tipping confirmation is shown with appropriate design
5. Confirm the Tweet share button works correctly
6. Confirm the transaction is recorded in `brave://rewards`
7. Redo steps 3 - 6, but set a monthly recurring tip

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
